setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="sac_utils",
+    version="0.0.1",
+    author="Satellite Applications Catapult",
+    #author_email="author@example.com",
+    description="A collection of utility functions for use in Jupyter Notebooks and other geo processing",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/SatelliteApplicationsCatapult/sac_utils",
+    packages=setuptools.find_packages(include=['sac_utils']),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
```

Comment:
is it worth pulling the requirements.txt file in here so this will be aware of the dependencies.